ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -122,13 +133,18 @@ internal class PaywallViewModelImpl(
     }
 
     override fun restorePurchases() {
+        if (_actionInProgress.value) { return }
```

Comment:
Maybe we should log something in these cases?

Reply:
Okay 👍🏻